js/samples/04.ai-apps/d.assistants-mathBot/src/bot.ts

```diff
@@ -1,21 +1,27 @@
 import { Application, preview, AI } from '@microsoft/teams-ai';
 import { MemoryStorage, TurnContext } from 'botbuilder';
 
-if (!process.env.OPENAI_KEY) {
-    throw new Error('Missing environment variables - please check that OPENAI_KEY.');
+if (!process.env.AZURE_OPENAI_KEY || !process.env.AZURE_OPENAI_ENDPOINT) {
+    throw new Error(
+        'Missing environment variables - please check that AZURE_OPENAI_KEY and AZURE_OPENAI_ENDPOINT is set.'
```

Comment:
Can we update this to either OAI / AOAI is not set

Reply:
it doesn't make sense to do that b.c. even if OpenAI key is set, the azure creds are used by default in line 16. Let me know what you think.

Reply:
I think we want the samples to run with either api key configured, so instead you could do something like `const apiKey = process.env.OPENAI_KEY || process.env.AZURE_OPENAI_KEY` and use that

Reply:
implemented - thanks for the suggestion @lilyydu